openedx/core/djangoapps/content/course_overviews/tests.py

```diff
@@ -485,3 +492,226 @@ def test_get_all_courses_by_org(self):
             {c.id for c in CourseOverview.get_all_courses(org='TEST_ORG_1')},
             {c.id for c in org_courses[1]},
         )
+
+
+@ddt.ddt
+class CourseOverviewImageSetTestCase(ModuleStoreTestCase):
+    """
+    Course thumbnail generation tests.
+    """
+
+    def setUp(self):
+        """Create an active CourseOverviewImageConfig with non-default values."""
+        CourseOverviewImageConfig.objects.create(
+            enabled=True,
+            small_width=200,
+            small_height=100,
+            large_width=400,
+            large_height=200,
+        )
+        super(CourseOverviewImageSetTestCase, self).setUp()
+
+    @override_settings(DEFAULT_COURSE_ABOUT_IMAGE_URL='default_course.png')
+    @override_settings(STATIC_URL='static/')
+    @ddt.data(
```

Comment:
@adampalay thought I'd point this out...not sure if this is the test pattern you were asking me about, but FYI